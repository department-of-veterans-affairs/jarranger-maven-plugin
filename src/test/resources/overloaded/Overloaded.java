@SuppressWarnings("all")
public final class Overloaded {
  public String getClientInfo(Double num) {
    return null;
  }

  public void setClientInfo(Object p) {}

  public int getHodlability() {
    return 0;
  }

  public void setClientInfo(Double num) {}

  public Object getClientInfo() {
    return null;
  }

  public void setClientInfo(String key, String value) {}

  public Object getClientInfo(Object obj) {
    return null;
  }

  public void setCatalog(String catalog) {}

  public String getClientInfo(String name) {
    return null;
  }
}
