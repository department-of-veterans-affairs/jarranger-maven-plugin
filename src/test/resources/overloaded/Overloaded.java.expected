@SuppressWarnings("all")
public final class Overloaded {
    public String getClientInfo(Double num) {
        return null;
    }

    public Object getClientInfo() {
        return null;
    }

    public Object getClientInfo(Object obj) {
        return null;
    }

    public String getClientInfo(String name) {
        return null;
    }

    public void setClientInfo(Object p) {
    }

    public void setClientInfo(Double num) {
    }

    public void setClientInfo(String key, String value) {
    }
    
    public int getHodlability() {
        return 0;
    }

    public void setCatalog(String catalog) {
    }
}
